telegram/ext/_application.py

```diff
@@ -1076,6 +1082,8 @@ async def _update_fetcher(self) -> None:
                     self.create_task(
                         self.__process_update_wrapper(update),
                         update=update,
+                        name=f"Application:{self.bot.id}:process_concurrent_update_"
+                        f"{self.update_queue.qsize()}",
```

Comment:
https://github.com/python-telegram-bot/python-telegram-bot/pull/3759#discussion_r1238700959 ? ;P

Reply:
oh my bad thought I removed it